samples/notebooks/KubeFlow Pipeline Using TFX OSS Components.ipynb

```diff
@@ -133,8 +145,13 @@
    "source": [
     "# Note that this notebook should be running in JupyterHub in the same cluster as the pipeline system.\n",
     "# Otherwise it will fail to talk to the pipeline system.\n",
+    "import kfp\n",
+    "from kfp import compiler\n",
+    "import kfp.dsl as dsl\n",
+    "import kfp.notebook\n",
+    "import kfp.gcp as gcp\n",
     "client = kfp.Client()\n",
-    "exp = client.create_experiment(name='demo')"
+    "exp = client.create_experiment(name=EXPERIMENT_NAME)"
```

Comment:
I think we should reuse the experiment if it already exists.

Reply:
Yes. But for that, we can have another PR to do that. The scope for this PR is to add the sample tests, and it requires parameterizing the pipeline notebooks.